payloads/library/credentials/PublicIPGrabber/payload.txt

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+LED B
+ATTACKMODE HID
+QUACK switch1/duck_code.txt

```

Comment:
You can use `GET SWITCH_POSITION` to export the switch position variable and then call it in your payload. 

```
GET SWITCH_POSITION
QUACK $SWITCH_POSITION/duck_code.txt
```